src/ecl/eclWatchTree.ts

```diff
@@ -421,7 +439,8 @@ export class ECLWUNode extends Item<ECLWatchTree> {
     }
 
     contextValue(): string {
-        return this._wu.isComplete() ? "ECLWUNodeComplete" : "ECLWUNode";
+        const prot = this._wu.Protected ? "Protected" : "Unprotected";
+        return this._wu.isComplete() ? `ECLWUNodeComplete${prot}` : `ECLWUNode${prot}`;
```

Comment:
Change to 
```ts
return this._wu.isComplete() ? `ECLWUNodeComplete,${prot}` : `ECLWUNode,${prot}`;
```
respectively

Reply:
Yes. I needed to simplify here after testing.